agent/installer/registry_test.go

```diff
@@ -36,21 +36,24 @@ var _ = Describe("Byohost Installer Tests", func() {
 			Expect(r.GetInstaller("a", "b")).To(BeNil())
 		})
 		It("Should allow working with installers", func() {
-			Expect(func() { r.AddBundleInstaller("ubuntu", "1.22", dummy122) }).NotTo(Panic())
-			Expect(func() { r.AddBundleInstaller("ubuntu", "1.23", dummy123) }).NotTo(Panic())
-			Expect(func() { r.AddBundleInstaller("rhel", "1.24", dummy124) }).NotTo(Panic())
+			Expect(func() { r.AddBundleInstaller("ubuntu", "v1.22.*", dummy122) }).NotTo(Panic())
+			Expect(func() { r.AddBundleInstaller("ubuntu", "v1.23.*", dummy123) }).NotTo(Panic())
```

Comment:
At this point we don't support `v1.23.*` and `v1.24.*`
These are negative test cases IMO. I understand the assertion is not to panic, but ideally anything other than `v1.22.*` should error out. Do we have such a test case?

Reply:
We don't have test like that, I will convert those to such. Thanks.